manifests/i/I-GIS/GeoScene3D/12.0.0.680/I-GIS.GeoScene3D.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: I-GIS.GeoScene3D
+PackageVersion: 12.0.0.680
+PackageLocale: en-US
+Publisher: I-GIS
+PackageName: GeoScene3D

```

```suggestion
PackageName: I-GIS GeoScene3D
```
This must match Control Panel.